### PR TITLE
feat: FIFS-174 Wrap/Unwrap

### PR DESCRIPTION
**Pull request by SerDorosh**
Wrap / Unwrap NEAR feature

### CODE REVIEW

**Comment by agileurbanite** [comment]
@SerDorosh we won't block this PR unless it introduces serious bugs but @gutsyphilip will leave a bunch of comments just for best practices

File: packages/frontend/src/components/wrap/SwapNear.js [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+                                    setSwappingToken('failed');
+                                    return;
+                                }
+                            );
+
+                            dispatch(checkAndHideLedgerModal());
+                        }}
+                        swappingToken={swappingToken}
+                    />
+                );
```
Comment (gutsyphilip) [inline packages/frontend/src/components/wrap/SwapNear.js:280]: We should extract this into it's own function.

File: packages/frontend/src/components/wrap/components/SwapAmountForm.js [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+                            const tempMax = maxTo;
+                            const tempToken = activeTokenTo;
+                            setActiveTokenTo(activeTokenFrom);
+                            setActiveTokenFrom(tempToken);
+                            setAmountTokenTo(amountTokenFrom);
+                            setAmountTokenFrom(temp);
+                            setMaxTo(maxFrom);
+                            setMaxFrom(tempMax);
+                            setMockRateData(1 / mockRateData);
+                        }}
```
Comment (gutsyphilip) [inline packages/frontend/src/components/wrap/components/SwapAmountForm.js:144]: Extract to named function.

File: packages/frontend/src/components/wrap/components/helper.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
+import { removeTrailingZeros } from '../../../utils/amounts';
+
+export const getFormatBalance = (num, decimals) => {
+    if (!num || num === '0') {
+        return {
+            numToShow: '0',
```
Comment (gutsyphilip) [inline packages/frontend/src/components/wrap/components/helper.js:6]: These utils should be moved to `src/utils/wrap-unwrap.js`

**Review by gutsyphilip — CHANGES_REQUESTED** [review]
Looks good overall. Amazing work!
Just requested a couple of changes to adhere to best practices and our standards.

**Comment by SerDorosh** [comment]
Thanks!
Done.

**Review by gutsyphilip — APPROVED** [review]
Nice work. Thanks for making the updates. 
Made some UI updates based on the design. 
I'll go ahead and approve this now.

**Pull request merged**